src/python/foglamp/admin_api/server.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+import asyncio
+from foglamp.admin_api import controller as admin_api_controller
+ 
+ 
+def start():
+    """Starts the service"""
+    admin_api_controller.start()
```

Comment:
First comment: The Python guide is currently not specific about how to import. I'll add to it.

Second comment: Yes, we should have both so that you can python -m foglamp.device instead of foglamp.device.server. server.py is for tests and other modules to use, __main__ is convenience for users.